clang/lib/Sema/SemaTemplateDeductionGuide.cpp

```diff
@@ -942,34 +946,145 @@ getRHSTemplateDeclAndArgs(Sema &SemaRef, TypeAliasTemplateDecl *AliasTemplate) {
   return {Template, AliasRhsTemplateArgs};
 }
 
+struct InheritedConstructorDeductionInfo {
+  // Class template for which we are declaring deduction guides
+  // This is `C` in the standard wording
```

Comment:
Comments are supposed to end in full-stops, so I think there is a sprinkling of `.` required here/elsewhere here (other than standards quotes).